apps/api/src/app/workflows-v2/usecases/sync-to-environment/sync-to-environment.usecase.ts

```diff
@@ -61,14 +58,13 @@ export class SyncToEnvironmentUseCase {
   private async buildRequestDto(
     originWorkflow: WorkflowResponseDto,
     preferencesToClone: PreferencesEntity[],
-    command: SyncToEnvironmentCommand,
     targetWorkflow?: WorkflowResponseDto
   ) {
     if (targetWorkflow) {
-      return await this.mapWorkflowToUpdateWorkflowDto(originWorkflow, targetWorkflow, preferencesToClone, command);
+      return this.mapWorkflowToUpdateWorkflowDto(originWorkflow, targetWorkflow, preferencesToClone);
```

Comment:
I really like having await waiting for the result of an async function, I saw you removed them in bulk, is it a technical reason or preference? I like it for readability, it's clear to me those are both promises but since you didn't define the retuen type it's a little obscure

Reply:
I would say its a preference, having `await` when returning async function inside async function is redundant - you can see that the function is a promise by hovering over that function.

![image](https://github.com/user-attachments/assets/a4a71665-b2fd-48b1-84b3-fc2e24f7d9eb)

Although the are some advantages to use redundant await when debugging. We can enforce and ESLint rule to enforce either way.

Reply:
If it's ok with you I would prefer to leave it there for now, lets add this stuff and have a standards call and decide on those stuff and enforce them by eslint